### ISSUE TITLE
Uglify seems not to do anything

### ISSUE THREAD

**Issue by jscti** [nobrainr/typescript-webpack-starter#19]
Hi there.

First, thanks for this starter kit, it was a great help.

From a fresh newly created project using your cli command, I updated the index.ts to add just a comment : 
```
export default class Main {
  constructor() {
    // Hello from the comments
    console.log("Typescript Webpack starter launched");

    if (/test/.test("test")) {
      console.log("hello there");
    }
  }
}

let start = new Main();
```

After a `npm run build`, If you inspect the app.bundle.js generated, you'll see that : 
- comments are not stripped
- code is not at all uglified
- there is a huge amount of source mappings

A tiny part of the generated file : 

> sourcesContent:['export default class Main {\n  constructor() {\n    // Hello from the comments\n    console.log("Typescript Webpack starter launched");\n\n    if (/test/.test("test")) {\n      console.log("hello there");\n    }\n  }\n}\n\nlet start = new Main();\n']}

**Comment by emyann** [comment]
Hello @jscti ! Thank you for reporting this issue and sorry for this delayed response. Indeed there was an issue in the way uglifyJs and istanbul was configured for the coverage. 
Those issues should have been fixed with the latest version of the CLI => **v0.3.0** https://www.npmjs.com/package/create-ts-lib 
I made a couple of tests in order to ensure most of the comments are stripped as well as the code to be well uglified.

Here are some bundle size comparisons of the project sample coming along with the CLI:
```sh
npm run build:dev # Output a developer bundle
```
<img width="493" alt="screen shot 2018-08-22 at 12 06 11 pm" src="https://user-images.githubusercontent.com/4748419/44477633-3510a580-a609-11e8-9241-160f81d46ce4.png">

```sh
npm run build # Output a production bundle, minified with tree shaken
```
<img width="489" alt="screen shot 2018-08-22 at 12 06 56 pm" src="https://user-images.githubusercontent.com/4748419/44477662-478adf00-a609-11e8-8059-cffa8bf7ac14.png">

Also, I personnaly use this CLI as a base scaffolder for this library https://bundlephobia.com/result?p=morphism. The improvements in the bundle size from the last 3 versions come from the tweak of this CLI. This is something I will keep on enhancing. 

Thanks and let me know if you find other improvements! 🙏🏽
